### PR TITLE
SV-11 camera switching 

### PR DESCRIPTION
**Pull request by hjed**
Basic camera switching capabilities.

Needs some cleanup mainly in terms of scaling (although I'd love to change the messages we use).

### CODE REVIEW

**Comment by hjed** [comment]
Actually don't review yet. Need to add header comments, etc

**Comment by hjed** [comment]
This is now ready for review

File: qt-gui.workspace.user.865c78c [hunk 1]
```diff
@@ -0,0 +1,308 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sajidanower23) [inline qt-gui.workspace.user.865c78c]: Should this file be tracked by git?

**Review by sajidanower23 — COMMENTED** [review]
LGTM. I will test run the code on my computer tonight

File: qt-gui.workspace.user.865c78c (hunk 1 above)
Comment (hjed) [reply qt-gui.workspace.user.865c78c]: Good question @sajidanower23 . Removed and added to .gitignore

**Review by sajidanower23 — APPROVED** [review]

**Pull request merged**